common/src/main/java/com/wynntils/functions/LootrunFunctions.java

```diff
@@ -121,6 +122,24 @@ public FunctionArguments.Builder getArgumentsBuilder() {
         }
     }
 
+    public static class LootrunMissionFunction extends Function<String> {
+        @Override
+        public String getValue(FunctionArguments arguments) {
+            int missionIndex = arguments.getArgument("index").getIntegerValue();
+
+            MissionType mission = Models.Lootrun.getMission(missionIndex);
+            if (mission == null) return "§cNone";
```

Comment:
We don't do coloring for function outputs 99% of the time, it just makes them more annoying to use, and less flexible.

Reply:
How would you suggest I color the Mission names on the Overlay?

Reply:
You can add a secondary argument for colored/not.